custom-namespaces.yaml

```diff
@@ -229,3 +229,13 @@ newtab:
   owners:
     - mmccorquodale@mozilla.com
   spoke: looker-spoke-private
+awesome_bar:
+  glean_app: false
+  owners:
+    - tbrooks@mozilla.com
+  pretty_name: Awesome Bar
+  views:
+    urlbar_clients_daily:
+      type: ping_view
+      tables:
+        - table: mozdata.telemetry.urlbar_clients_daily
```

Comment:
@teonbrooks Are there any other tables you'd need?

Reply:
@scholtzan nope, that's it :)

Reply:
just curious, i see that the type is `ping_view`, what does that mean?